fixup autotool warning:

configure.ac:661: warning: AC_COMPILE_IFELSE was called before AC_GNU_SOURCE
autoconf/specific.m4:335: AC_GNU_SOURCE is expanded from...
aclocal.m4:8669: gl_LOCK_BODY is expanded from...
aclocal.m4:8463: gl_LOCK is expanded from...
aclocal.m4:609: gt_INTL_SUBDIR_CORE is expanded from...
aclocal.m4:511: AM_INTL_SUBDIR is expanded from...
aclocal.m4:387: AM_GNU_GETTEXT is expanded from...
configure.ac:661: the top level
configure.ac:661: warning: AC_RUN_IFELSE was called before AC_GNU_SOURCE

--- configure.ac
+++ configure.ac
@@ -57,6 +57,7 @@
 AC_PROG_MAKE_SET
 dnl AC_PROG_RANLIB
 AC_PROG_YACC
+AC_GNU_SOURCE
 
 AC_AIX
 AC_MINIX
